### DISCUSSION TITLE
Links on docs

### DISCUSSION THREAD

**Discussion by AWS-crafter** [PGScatalog/pgsc_calc#190] [General]
I know this was updated today, so apologies if this was known already.

The link:
https://ftp.ebi.ac.uk/pub/databases/spot/pgs/resources/pgsc_1000G_v1.tar.zst
at this location: https://pgsc-calc.readthedocs.io/en/latest/how-to/ancestry.html#ancestry
exists.

The link:
https://ftp.ebi.ac.uk/pub/databases/spot/pgs/resources/pgsc_HGDP+1kGP_v1.tar.zst
at this location: https://pgsc-calc.readthedocs.io/en/latest/how-to/ancestry.html#ancestry
does not exist.

The link: https://pgsc-calc.readthedocs.io/en/latest/how-to/database.html#database
at this location: https://pgsc-calc.readthedocs.io/en/latest/how-to/database.html#database
does not exist. (It used to, so presumably the link should be the first working link instead.)

No bugs or anything--just figured I'd post this just in case.

**Comment by AWS-crafter** [comment]
Oops. I noticed this was already noticed here: https://github.com/PGScatalog/pgsc_calc/issues/183

**Comment by nebfield** [comment]
We found some problems with the new reference database during testing so we re-published it this morning 😅 Thanks for letting us know 🚀
